lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -86,88 +84,37 @@ def paintEvent(self, e):
         if not hasattr(self, 'renderer'):
             return
 
-        if DEBUG:
-            print('FigureCanvasQtAgg.paintEvent: ', self,
-                  self.get_width_height())
-
-        if len(self.blitbox) == 0:
-            # matplotlib is in rgba byte order.  QImage wants to put the bytes
-            # into argb format and is in a 4 byte unsigned int.  Little endian
-            # system is LSB first and expects the bytes in reverse order
-            # (bgra).
-            if QtCore.QSysInfo.ByteOrder == QtCore.QSysInfo.LittleEndian:
```

Comment:
fair.  There are not many little-endian systems left in practice (and they tend to be big main frame type systems these days).